src/utils/customMessages.js

```diff
@@ -61,7 +61,10 @@ const messages = {
   invalidRole: 'The role must be Super Administrator, Travel Administrator, Travel Team Member, Manager or Requester',
   existingRole: 'That is the existing role for this user',
   superUser: 'You can not change the role for the super user',
-  noTwoSuperUser: 'You can not'
+  noTwoSuperUser: 'You can not',
+  requestsRetrieved: 'Requests have been retrieved successfully',
+  noRequestsYet: 'You don\'t have any request, you can create some',
```

Comment:
could you not use contractions in messages. For instance, you could use the formal version of **don't** which is **do not**

Reply:
unfortnately these are not my codes

Reply:
took care of that